jsonnet/telemeter/metrics.jsonnet

```diff
@@ -51,4 +51,8 @@
   // cluster alerting for the purposes of preventing upgrades from
   // failing for end users.
   '{__name__="ALERTS",alertstate="firing"}',
+  // the following three metrics will be used for SLA analysis reports.
+  '{__name__="code:apiserver_request_count:rate:sum"}',
+  '{__name__="kube_pod_status_ready",condition="true",namespace="kube-system",pod=~"etcd.*"}',
```

Comment:
if this is something we want to report, then it should be pre-aggregated in a recording rule, so we only ship the sum of pods in a state instead of a per pod cardinality

Reply:
I will make this change and put a PR into the monitoring operator